client/hotrod-client/src/main/java/org/infinispan/client/hotrod/event/ClientListenerNotifier.java

```diff
@@ -385,6 +386,10 @@ private ClientListenerInvocation(Object listener, Method method) {
       public void invoke(ClientEvent event) {
          try {
             method.invoke(listener, event);
+         } catch (InvocationTargetException e) {
+            log.errorInvokingListener(method, listener, e.getTargetException());
+            throw log.exceptionInvokingListener(
```

Comment:
Isn't it enough to throw the logging exception instead of also logging locally ?